e2e/kas-grants.bats

```diff
@@ -235,4 +235,22 @@ teardown_file() {
     run_otdfctl_kasg unassign $ATTR_ID_FLAG $NS_ID_FLAG $KAS_ID_FLAG
       assert_failure
       assert_output --partial "Must specify exactly one Attribute Namespace ID, Definition ID, or Value ID to unassign"
+}
+
+@test "optional ID flag string error message" {
+    export NS_ID_FLAG='--namespace-id hello'
+    export ATTR_ID_FLAG='--attribute-id world'
+    export VAL_ID_FLAG='--value-id goodnight'
+
+    run_otdfctl_kasg unassign $NS_ID_FLAG $KAS_ID_FLAG
+      assert_failure
+      assert_output --partial "Optional flag '--namespace-id' received value 'hello' and must be a valid UUID if used"
```

Comment:
At some future point we should have some utility that exports all our error strings into a bats file so we can make these assert_output more durable.